sw/device/lib/testing/flash_ctrl_testutils.c

```diff
@@ -169,7 +169,8 @@ bool flash_ctrl_testutils_write(dif_flash_ctrl_state_t *flash_state,
                                 dif_flash_ctrl_partition_type_t partition_type,
                                 uint32_t word_count) {
   // Cannot program partial flash words.
-  if (byte_address & (FLASH_CTRL_PARAM_BYTES_PER_WORD - 1)) {
+  // TODO: #13773 for more details
+  if (byte_address & sizeof(uint32_t) - 1) {
```

Comment:
What will happen if the region was scrambled or enabled ecc? Will retval show some error or is this silent? If you know, it would be good to add the info in the function description in the header file.